external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -8,23 +8,27 @@ This page discusses how to configure your cube with the greatest control and fle
 The `qube-config.json` file has two sections:
```

Comment:
This is just a personal thing that maybe only applies to me, but as someone looking at this for the first time I thought I might find it useful to either mention or link to the schema/its location more directly, do you think that might belong here or in the "writing a qube-config.json" guide that will be created in the future?

Reply:
Thanks, have added this as a suggestion in the new ticket (#706)